### PR TITLE
CXG-233 gene mapping

### PR DESCRIPTION
**Pull request by jahilton**
Add documentation and script on how to evaluate gene mappings across GENCDOE versions

### CODE REVIEW

**Comment by Jchaffer787** [comment]
LGTM!

File: cellxgene_resources/curation_sample_code.ipynb [hunk 1]
```diff
@@ -849,6 +849,23 @@
...
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "v44_gene_map = json.load(open('../gene_ID_mapping/gene_map_v44.json'))\n",
+    "adata.var.rename(index=v44_gene_map, inplace=True)"
```
Comment (jychien) [inline cellxgene_resources/curation_sample_code.ipynb:866]: Jim had put in an extra check to see that the newly mapped ID is not already in the index.

**Comment by jychien** [comment]
For gene_mapping.py, I looked it over but it was too complicated and couldn't following everything. But we've thoroughly gone over the output to this script, so I think that's good for the review.

File: cellxgene_resources/curation_sample_code.ipynb (hunk 1 above)
Comment (jahilton) [reply cellxgene_resources/curation_sample_code.ipynb]: Good catch. Please review the added line

Comment (jychien) [reply cellxgene_resources/curation_sample_code.ipynb]: Looks good!

**Pull request merged**